tests/slow/src/test/scala/tests/feature/WorksheetCrossLspSuite.scala

```diff
@@ -2,5 +2,4 @@ package tests.feature
 
 import scala.meta.internal.metals.{BuildInfo => V}
 
-object Worksheet211LspSuite extends tests.BaseWorksheetLspSuite(V.scala211)
```

Comment:
Instead of removing these tests for all JVM versions, could we skip them on newer java versions?

Reply:
Yes. I've updated the code to discriminate on the JDK version and Scala versions.